processFlows.dscp

```diff
@@ -34,12 +43,24 @@ token IssuedCert {
 pub fn initiate_cert || => |output: InitiatedCert| where {
   output.hydrogen_owner == sender,
   output.energy_owner != sender,
+  output.regulator != sender,
 }
 
 pub fn issue_cert |input: InitiatedCert| => |output: IssuedCert| where {
   output == input,
   input.hydrogen_owner == output.hydrogen_owner,
   input.energy_owner == output.energy_owner,
```

Comment:
still missing `input.regulator == output.regulator` on this flow

Reply:
updated and thank for a flag up